articles/active-directory/develop/quickstart-v2-java-webapp.md

```diff
@@ -134,9 +165,9 @@ You can add the reference to MSAL4J by adding the following code to the top of t
 import com.microsoft.aad.msal4j.*;
 ```
 
-## Next Steps
+## More Related Information
```

Comment:
I think Next Steps works better here, as it follows the same pattern as other quickstarts

Reply:
Yes Next Steps header is the standard structure recommended by the Docs platform so let us keep that.

Reply:
Changed it back to "Next Steps"